### PR TITLE
Add vcrun6sp6

### PR DESCRIPTION
**Pull request by Zemogiter**
Needed for The Sims 3. Ready for review.

### CODE REVIEW

File: Engines/Wine/Verbs/vcrun6sp6/script.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+     var setupFile = new CabExtract()
+        .archive(toBeCabExtracted)
+        .to(this.prefixDirectory() + "/drive_c/vcrun6sp6/")
+        .extract(["-L", "-F", "vcredist.exe"]);
+
+    remove(this.system32directory() + "atl80.dll");
+    remove(this.system32directory() + "msvcm80.dll");
+    remove(this.system32directory() + "msvcp80.dll");
+    remove(this.system32directory() + "msvcr80.dll");
+    remove(this.system32directory() + "vcomp.dll");
```
Comment (ImperatorS79) [inline Engines/Wine/Verbs/vcrun6sp6/script.js]: Those are not the dlls removed by winetricks:
```
w_try rm -f "$W_SYSTEM32_DLLS"/comcat.dll
w_try rm -f "$W_SYSTEM32_DLLS"/msvcrt.dll
w_try rm -f "$W_SYSTEM32_DLLS"/oleaut32.dll
w_try rm -f "$W_SYSTEM32_DLLS"/olepro32.dll
w_try rm -f "$W_SYSTEM32_DLLS"/stdole2.tlb
```

File: Engines/Wine/Verbs/vcrun6sp6/script.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    this.wizard().wait(tr("Please wait while {0} is installed...", "vcrun6sp6"));
+    this.run(setupFile, "/q", null, false, true);
+
```
Comment (ImperatorS79) [inline Engines/Wine/Verbs/vcrun6sp6/script.js]: This call is missing:
```
w_try_cabextract vcredist.exe -d "$W_SYSTEM32_DLLS" -F mfc42u.dll
```

Comment (Zemogiter) [reply Engines/Wine/Verbs/vcrun6sp6/script.js]: That call is already in mfc42 verb.

Comment (ImperatorS79) [reply Engines/Wine/Verbs/vcrun6sp6/script.js]: OK then.

**Pull request merged**